contracts/DaoV2.sol

```diff
@@ -577,6 +578,13 @@ contract Dao {
             return _SYNTHFACTORY;
         }
     }
+    function RESERVE() public view returns(iRESERVE){
+        if(daoHasMoved){
+            return Dao(DAO).RESERVE();
+        } else {
+            return _RESERVE;
+        }
+    }
```

Comment:
Handle Reserve address for public view where the DAO has changed the Reserve's address